content/webapp/utils/iiif/v3/index.ts

```diff
@@ -240,7 +240,7 @@ export function getImageAuthProbeService(
   return Array.isArray(service)
     ? service?.find(s => s.type === 'AuthProbeService2')
     : service?.type === 'AuthProbeService2'
-      ? service
+      ? (service as unknown as AuthProbeService2)
```

Comment:
@gestchild I could use your knowledge here; `BodyService2` and `AuthProbeService2` don't match up, which I had to tell TS to ignore here. Should the type of service be `AuthProbeService2` | undefined? Let's chat!

Reply:
The typing is a bit hacky here to be honest and probably need revisiting properly in a separate PR. I think this is fine as a solution as if we return something here we know the service type is AuthProbeService2